core/src/test/scala/org/apache/spark/LocalSparkContext.scala

```diff
@@ -27,12 +27,12 @@ trait LocalSparkContext extends BeforeAndAfterEach with BeforeAndAfterAll { self
 
   @transient var sc: SparkContext = _
 
-  override def beforeAll() {
+  override def beforeAll(): Unit = {
```

Comment:
I think you could take it out of this PR. There are so many instances of this syntax that I'm not even sure it's worth changing everywhere. It's fine to fix if you're already modifying the surrounding code, but this is unrelated